lib/kvdb/kvdb_rparams.c

```diff
@@ -977,6 +977,27 @@ static const struct param_spec pspecs[] = {
             },
         },
     },
+    {
+        .ps_name = "durability.size_bytes",
```

Comment:
I would say just looking at the description, I am asking myself "size of what". Size of durability doesn't make much sense in english to me.

Reply:
Durability size indicates the number of bytes that can be lost in the event of a crash. I will change the description. This is an experimental parameter and don't have plans to expose this for an user.

Reply:
Now it makes more sense. Thanks